### ISSUE TITLE
debian-base: Disable accessibility bus

### ISSUE THREAD

**Issue by matthiasbock** [matthiasbock/recipes#10]
To suppress error messages like
~~~
Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-xxfluS2Izg: Connection refused
~~~

in .bashrc add:
~~~
export NO_AT_BRIDGE=1
~~~

Source: https://unix.stackexchange.com/questions/230238/x-applications-warn-couldnt-connect-to-accessibility-bus-on-stderr
